.gitignore

```diff
@@ -0,0 +1,32 @@
+# Ignore Ansible retry files
+*.retry
+
+# Ignore temporary files
+*.tmp
+*.bak
+*.swp
+*.swo
+
+# Ignore roles directory except requirements.yml
+ansible/roles/**
+!roles/requirements.yml
+
+# Ignore inventory files
+# ansible/inventory/ # Not for now, but maybe in the future....
```

Comment:
As we are not saving any IP address, port os minimal sensitive data, we are committing this file as a way of documentation.